app/assets/javascripts/administrate/components/belongs_to_form.js

```diff
@@ -0,0 +1,23 @@
+$(function() {
+  $(".field-unit--belongs-to select[data-remote-url]").each(function(index, field) {
+    $(field).selectize({
+      valueField: 'id',
+      labelField: 'name',
+      searchField: 'name',
+      preload: 'focus',
+      load: function(query, callback) {
+          if (!query.length) return callback();
```

Comment:
So the problem with that is that dropdown is blank unless user starts typing which is not what we want: in case user does not have a particular record in mind he won't be able to select anything.

I'm not sure what would be good UX here as a start I'd recommend:
1. Pressing `keydown` should populate select with all records
2. Pressing an arrow on the dropdown should populate select with all records

Reply:
I'm not the author (obviously), but the PR is to resolve huge result sets bringing the browser down because the whole result set is loaded into a dropdown.

If you have 100,000 users, you're going to run into the same problem any way you fill the dropdown with all records.

Maybe it would be worth making this configurable somehow - so being able to turn on the remote search / query?